src/Workspaces/VisualBasicTest/Formatting/FormattingTests.vb

```diff
@@ -902,11 +902,11 @@ End Class"
             Dim expected = $"Class C
     Sub Method(Optional ByVal i As Integer = 1)
         Dim aa = 1 + {continuation}
- {continuation}
- {continuation}
- {continuation}
- {continuation}
- {continuation}
+                     {continuation}
```

Comment:
💭 I'm not a fan of aligning these to a variable column. Any of the following would be an improvement:

* Always using the first column (i.e. the behavior prior to this change)
* Always using the same column as the _start_ of the previous line (e.g. align to the `D` in `Dim`)
* Always using a fixed indent relative to the start of the containing statement (e.g. indent by 4 spaces relative to `Dim`)

Reply:
I'll add this discussion to the design review queue for next week.

Reply:
Thanks, I am open 2nd or 3rd option, the original design is really hard to read which is why I did this.

Reply:
@sharwell Any updates?

Reply:
➡️ This formatting is fine per https://github.com/dotnet/roslyn/pull/54559#issuecomment-886767236